using-mobile-extensions/mobile-core/rules-engine/rules-engine-consequence-details.md

```diff
@@ -0,0 +1,83 @@
+# Rules engine consequence detail definition
+
+The Adobe Experience Platform Mobile SDK supports multiple types of rule consequences (also known as actions) that can be executed when the trigger event and the conditions are met. This section contains a detailed description of each rule consequence.
+
+## Analytics consequence
+
+This rule consequence is currently handled by the [Analytics](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-analytics) extension.
+
+| Friendly name   | Key         | Type   | Description                                                  |
+| --------------- | ----------- | ------ | ------------------------------------------------------------ |
+| Action          | action      | string | *(Optional)* If provided, this value will be used as the action parameter in a `trackAction` call. |
+| State/Page name | state       | string | *(Optional)* If provided, this value will be used as the state parameter in a `trackState` call. |
+| Context data    | contextdata | object | *(Optional)* Additional context data to be attached to the resulting Analytics request. The object should only have one level of depth, that contains <string, string> key-value pairs. |
```

Comment:
In the 2nd description sentence, please remove the comma between "depth" and "that".